### ISSUE TITLE
Can't create new board on mac

### ISSUE THREAD

**Issue by RasmusFonseca** [GGBRW/BOOLR#18]
Menu opens up when I launch and I press 'new board'. I then type "testboard" and click "Create", but nothing happens.

**Comment by geolessel** [comment]
Same for me. I also tried doing the `ctrl-click` tip to open a sandbox board. Nothing.

**Comment by GGBRW** [comment]
Yes, very strange! I heard that a couple of times before and I have no idea why. Deleting the folder and downloading it again helps most of the time..

**Comment by terribleplan** [comment]
This happens because the .app packaging/sandboxing on macOS does not allow editing of the contents (which the application tries to do by saving a file for your board). For some reason once you drop it into your Applications directory something about paths or sandboxing changes and it will work.

This is a [known limitation](https://github.com/electron/electron/issues/7402). The workaround is to only write files to `app.getPath('appData') `

**Comment by readjfb** [comment]
I've also been having this problem, and I moved the application into the Applications directory, but it still won't let me create a new board. Can someone elaborate on how to fix this?
Thanks!

**Comment by RKD204TX** [comment]
After extracting, open the package contents of Boolr application, there you will find a MacOS folder which contains an unix executable file open it, it will run
